compile.sh

```diff
@@ -67,8 +73,19 @@ if [[ ! -f $SRC/.ignore_changes ]]; then
 	CHANGED_FILES=$(git diff --name-only)
 	if [[ -n $CHANGED_FILES ]]; then
 		echo -e "[\e[0;35m warn \x1B[0m] Can't update since you made changes to: \e[0;32m\n${CHANGED_FILES}\x1B[0m"
-		echo -e "Press \e[0;33m<Ctrl-C>\x1B[0m to abort compilation, \e[0;33m<Enter>\x1B[0m to ignore and continue"
-		read -r
+		while true; do
+			echo -e "Press \e[0;33m<Ctrl-C>\x1B[0m to abort compilation, \e[0;33m<Enter>\x1B[0m to ignore and continue, \e[0;33mdiff\x1B[0m to display changes"
```

Comment:
small detail, but the exit command is not described.

Reply:
Fixed ("exit" description).